tests/test_repros.py

```diff
@@ -909,8 +911,7 @@ def test_maml_no_item_capture(self):
                 self.assertTrue(same(model(a, b, c, d), correct))
 
         self.assertEqual(cnt.frame_count, ifdyn(5, 4))
-        # TODO(jansel): figure out why op count depends on imports
-        self.assertIn(cnt.op_count, (36, 35, 29, 28))
+        self.assertEqual(cnt.op_count, ifdyn(35, 28))
```

Comment:
Since functorch plans to the monkeypatch causing this issue, perhaps we should leave this the way it is then remove it once functorch changes.